src/checks.py

```diff
@@ -501,14 +501,25 @@ def check_build_and_test(state: State) -> Optional[str]:
 
     strategies = [BuildAndTestMaven(), BuildAndTestNpm()]
     errors: List[str] = []
+    executed_at_least_one = False
 
     for strategy in strategies:
         if not strategy.should_run(state):
             continue
+        executed_at_least_one = True
+        print(f"Executing build-and-test for {strategy.name()}")
         err = strategy.run(state)
         if err is not None:
             errors += f"{strategy.name()}: {err}"
 
+    if not executed_at_least_one:
+        errors = [
+            "Heuristics failed to figure out the way to build and test this release.",
```

Comment:
the word error concerns me. remember source releases do not have a requirement to have tests even..
  (such as the case with zipkin-api, even if we fake it by saying a linter is a test.)

I'm concerned we are getting ahead of ourselves and this will cause fewer people to do source release verification. For example, we only have 2 votes on zipkin-api and I'm fairly confident the reason for that is the scary output we have in the tooling. This happens to be our most important thing to release right now..

Reply:
instead, I'd suggest declassifying from error to notice. Ex notice: "this source release does not seem to include a build or test step." Then, whenever we have a .apacheverification file, we can explicitly squash that warning so that it doesn't scare people off from helping vote.